logger.gd

```diff
@@ -6,36 +6,60 @@
 
 extends Node  # Needed to work as a singleton
 
+
 ##================##
 ## Inner classes  ##
 ##================##
+class ExternalSink:
+	var name
+	var queue_mode = QUEUE_NONE
+	var buffer = PoolStringArray()
+	var buffer_idx = 0
```

Comment:
Is there a reason for this to be in `ExternalSink` instead of `Logfile`? It's not used in the "virtual" API provided by this base class so it's maybe best to leave it up to implementers to define how they want to handle their data to fit the API?

Reply:
I though every external sink should have a buffer / cache mechanism and so it will be universal usable. But if you think it is not necessary.